tree/ntuple/v7/test/ntuple_storage_daos.cxx

```diff
@@ -3,21 +3,31 @@
 #include "ROOT/TestSupport.hxx"
 #include <iostream>
 
+int destroy_container(const std::string& containerLabel) {
+
+//   daos_handle_t poolHandle{};
+//   daos_pool_connect(R__DAOS_TEST_POOL, nullptr, DAOS_PC_RW, &poolHandle, nullptr, nullptr);
+   return 0; //daos_cont_destroy(poolHandle, containerLabel.c_str(), 1, nullptr);
+}
+
```

Comment:
I would argue against directly using the libdaos API here.  I think it would actually be preferable to drop the container using a `daos container destroy` command issued by CMake after running the tests (and only if not libdaos_mock).

Reply:
I'll leave this change for an upcoming PR